lib/charliecloud.py

```diff
@@ -22,29 +22,25 @@
 
 ## Imports not in standard library ##
 
-# These are messy because we need --version and --help even if a dependency is
-# missing. Among other things, nothing can depend on non-standard modules at
-# parse time.
+sys.path.insert(0, (  os.path.dirname(os.path.abspath(__file__))))
```

Comment:
I think maybe the executables already do this?

Reply:
This was actually already here before I changed it to something else then back to that.

Reply:
I will double check my facts on the above, but I think it needs to be here.